docs/docs/References/Glossary.md

```diff
@@ -104,6 +104,9 @@ The process by which the Delphix Engine ingests data from a dataset on a [source
 ## Target Environment
 An [environment](#environment) on which Delphix-provided virtualized datasets can be used.
 
+## Lua Toolkit
+Legacy model for writing "plugins", in Lua with limited documentation and support for writing, building and uploading toolkits. This was the predecessor to the Virtualization SDK. 
```

Comment:
Minor nit: I think this would be clearer with a comma after "Lua".  As written, it seems like "in Lua with limited documentation and support for writing" is a standalone clause.

Reply:
fixed